src/hooks/useGestures.ts

```diff
@@ -65,15 +68,16 @@ const useGestures = (
         setGestureType(gesture);
       }
     }
-  };
+  }, []);
 
-  const touchMoveGesture = (e: any): void => {
+  const touchMoveGesture = useCallback(
+      (e: any): void => {
     if (!swipeTarget) {
       return;
     }
 
     e.preventDefault();
-  };
+  }, []);
 
   const attachGestures = (): void => {
```

Comment:
These functions may be removed and the listeners placed directly in a useMemo instead of useEffect.